python/cudf/cudf/core/dtypes.py

```diff
@@ -551,18 +791,51 @@ def __hash__(self):
         return hash(self._typ)
 
 
+@doc_apply(
+    decimal_dtype_template.format(
+        size="32",
+        example="""
+        >>> import cudf
+        >>> decimal32_dtype = cudf.Decimal32Dtype(precision=9, scale=2)
+        >>> decimal32_dtype
+        Decimal32Dtype(precision=9, scale=2)
+        """,
+    )
+)
 class Decimal32Dtype(DecimalDtype):
     name = "decimal32"
     MAX_PRECISION = np.floor(np.log10(np.iinfo("int32").max))
     ITEMSIZE = 4
 
 
+@doc_apply(
+    decimal_dtype_template.format(
+        size="64",
+        example="""
+        >>> import cudf
+        >>> decimal64_dtype = cudf.Decimal64Dtype(precision=15, scale=3)
+        >>> decimal64_dtype
+        Decimal64Dtype(precision=15, scale=3)
+        """,
+    )
+)
 class Decimal64Dtype(DecimalDtype):
     name = "decimal64"
     MAX_PRECISION = np.floor(np.log10(np.iinfo("int64").max))
     ITEMSIZE = 8
 
 
+@doc_apply(
+    decimal_dtype_template.format(
+        size="128",
+        example="""
+        >>> import cudf
+        >>> decimal128_dtype = cudf.Decimal128Dtype(precision=32, scale=4)
+        >>> decimal128_dtype
+        Decimal128Dtype(precision=32, scale=4)
+        """,
```

Comment:
The examples all look identical, I think you could inline those and just use `size` wherever you need them right? Of course, the allowed precisions change but no need to actually vary them in the examples.

Reply:
Resolved 😉